modules/es/fsw/src/cfe_es_task.c

```diff
@@ -1088,8 +1088,8 @@ int32 CFE_ES_QueryAllCmd(const CFE_ES_QueryAllCmd_t *data)
         OS_close(FileDescriptor);
         CFE_ES_Global.TaskData.CommandCounter++;
         CFE_EVS_SendEvent(CFE_ES_ALL_APPS_EID, CFE_EVS_EventType_DEBUG,
-                          "App Info file written to %s, Entries=%d, FileSize=%d", QueryAllFilename, (int)EntryCount,
-                          (int)FileSize);
+                          "App Info file written to %s, Entries=%d, FileSize=%lu", QueryAllFilename, (int)EntryCount,
```

Comment:
This could be `%zu` and then you wouldn't have to cast `FileSize` at all.

Reply:
Problem is not all "c99" compilers recognize %zu yet.... Once we update to C11 then we should be able to use the z modifier.   For now I prefer to keep the cast to unsigned long.